cranelift/filetests/filetests/isa/s390x/atomic_rmw.clif

```diff
@@ -573,7 +573,7 @@ block0(v0: i64, v1: i64, v2: i32):
 ; block0:
 ;   l %r0, 0(%r3)
 ;   0: clr %r4, %r0 ; jgnh 1f ; cs %r0, %r4, 0(%r3) ; jglh 0b ; 1:
-;   lgr %r2, %r0
+;   lr %r2, %r0
```

Comment:
@uweigand would you mind checking these move changes? They all seemed reasonable to me, as the result of each function is `i32`.